src/components/jobseekerchats/chats-middle.jsx

```diff
@@ -1,165 +1,66 @@
                 <button
-                  onClick={handleAccept}
-                  className="border-[#16730F] border-2 text-[#16730F] px-8 py-2 rounded-md  transition"
+                  type="button"
+                  className="bg-[#6B8E23] py-2 text-xs text-white w-full rounded-3xl hover:bg-[#5a7720] transition"
+                  aria-label="View Profile"
                 >
-                  Accepted
-                </button>
-                <button className="border-[#EB5757] border-2 text-[#EB5757] px-8 py-2 rounded-md  transition">
-                  Expired
+                  View Profile
                 </button>
               </div>
+            </div>
+          </div>
+  
+          {/* Contact + Links */}
+          <div className="bg-[#16730F] px-6 py-6 space-y-6 h-full">
+            {[
+              { label: "Email", value: "osakweprsca@gmail.com" },
+              { label: "Phone", value: "+234 7061410614" },
+              { label: "Website", value: "https://linktree.com" },
+            ].map((item, idx) => (
+              <div key={idx}>
+                <hr className="border-[#6B8E23] mb-2" />
+                <p className="text-white text-sm break-words">{item.value}</p>
               </div>
-              
-           
+            ))}
           </div>
-        </div>
-      </div>
-
-       {/* Message Input */}
-  <div className="p-4 bg-gray-100 ">
-    <div className="flex flex-col gap-2 border border-gray-300 rounded-2xl px-4 py-3 bg-gray-100 shadow-sm">
-      <input
-        type="text"
-        value={message}
-        onChange={(e) => setMessage(e.target.value)}
-        placeholder="Type a message"
-        className="flex-1 outline-none text-sm bg-transparent placeholder-gray-400"
-      />
-      <div className="flex justify-between items-center">
-        <div className="flex items-center space-x-2">
-          <button
-            type="button"
-            className="text-gray-500 hover:text-green-600 text-lg"
-            aria-label="Add emoji"
-          >
-            😊
-          </button>
-          <button
-            type="button"
-            className="text-gray-500 hover:text-green-600 text-lg"
-            aria-label="Attach file"
-          >
-            ＋
-          </button>
-        </div>
-        <div className="flex items-center space-x-2">
-          <button
-            type="button"
-            className="text-gray-500 hover:text-green-600 text-lg"
-            aria-label="Record voice"
-          >
-            🎤
-          </button>
-          <button
-            type="button"
-            className="bg-gray-700 hover:bg-green-600 text-white rounded-full p-2 transition"
-            aria-label="Send message"
-          >
-            ➤
-          </button>
-        </div>
-      </div>
-        </div>
+        </aside>
       </div>
-    </main>
-  );
+    );
 }
 
-export default ChatsMiddle;
\ No newline at end of file
+export default ChatsRight;
```

Comment:
Looks like a lot of updates are happening on this file. Please be sure this is really needed